station-dispenser-api-v0/build.gradle.kts

```diff
@@ -0,0 +1,9 @@
+import net.modificationstation.stationapi.gradle.SubprojectHelpers.addModuleDependencies
+import net.modificationstation.stationapi.gradle.SubprojectHelpers.getSubprojectVersion
+
+base.archivesName.set("station-dispenser-api-v0")
```

Comment:
Instead of creating a new module, this API should probably be a part of `station-blocks-v0` instead. Up for discussion.

Reply:
that's definitely a big change, I'll come back around to this but it's a lot of files and messy for now. For sure would be willing to consider after i fix everything else.